.dep-versions

```diff
@@ -15,4 +15,4 @@ pennylane=0.39.0.dev30
 # 'runtime/Makefile' and at all GitHub workflows, using the exact
 # commit hash corresponding to the merged PR that implements the
 # desired feature.
-lightning=0.39.0-dev38
+lightning=0.39.0-dev49
```

Comment:
Remember to update this when the lightning PR gets merged!

Same for git tag.

Reply:
Done